src/components/RepoList/actions.ts

```diff
@@ -2,15 +2,21 @@
 
 import { createClient } from "@/lib/supabase/server"
 import { revalidatePath } from "next/cache"
-import { redirect } from "next/navigation"
 
-export async function handleDelete(formData: FormData) {
-  "use server"
+export async function handleDelete(id: string) {
+  const { data, error } = await createClient()
+    .from("projects")
+    .delete()
+    .eq("id", id)
+    .select()
```

Comment:
If you want to receive one element only you should add `.single()`. This also updates the type of the `data` to be a single element and not an array.

Reply:
Great catch!